src/main/java/com/atlassian/maven/plugin/clover/CloverReportMojo.java

```diff
@@ -300,7 +300,7 @@ public void execute() throws MojoExecutionException {
     }
 
     @Override
-    public void generate(org.codehaus.doxia.sink.Sink sink, Locale locale) throws MavenReportException {
+    public void generate(org.apache.maven.doxia.sink.Sink sink, Locale locale) throws MavenReportException {
         executeReport(locale);
     }
```

Comment:
```suggestion
    // @Override till maven-reporting-api 3.0
    public void generate(org.codehaus.doxia.sink.Sink sink, Locale locale) throws MavenReportException {
        executeReport(locale);
    }

    // @Override since maven-reporting-api 3.1.0
    public void generate(org.apache.maven.doxia.sink.Sink sink, Locale locale) throws MavenReportException {
        executeReport(locale);
    }
```